jekyll/_cci2/android-images-support-policy.adoc

```diff
@@ -0,0 +1,94 @@
+---
+contentTags:
+  platform:
+  - Cloud
+  - Server v4.x
+  - Server v3.x
+---
+= CircleCI Android images support policy
+:page-layout: classic-docs
+:page-liquid:
+:page-description: CircleCI Android images release, update, and deprecation policy
+:icons: font
+:experimental:
+
+[#overview]
+== Overview
+
+This document outlines the Circleci Android image release, update, and deprecation policy. This policy applies to all CircleCI Android images built for the Machine executor.
+
+[#release-policy]
+== Release policy
+
+Android images are released once a quarter, with patch releases potentially being made for security issues. These images are generally built on top of the latest version of our most recent stable base Ubuntu image with added packages for Android.
+
+We install the most up-to-date versions of each tool/package in newly built images.
```

Comment:
Is there a way to make these bullet points like in doc?

Reply:
will do